### PR TITLE
Update lang-FR.yml

### PR DESCRIPTION
**Pull request by Nogapra**
Maj FR tard of Pet Master 1.12.2

### CODE REVIEW

File: src/main/resources/lang-FR.yml [hunk 1]
```diff
@@ -1,57 +1,69 @@
...
+player-offline: "Le joueur spécifié est hors ligne !"
+cannot-change-to-yourself: "Vous êtes déjà le propriétaire !"
+owner-changed: "Cet animal a changé de propriétaire !"
+change-owner-price: "Vous avez dépensé : AMOUNT !"
+new-owner: "Le joueur PLAYER vous a donné un animal !"
+not-owner: "Cet animal ne vous appartient pas !"
+right-click: "Faites clic-droit sur un animal pour changer son propriétaire !"
+pet-freed: "Dîtes adieu : cet animal est retourné à l'état sauvage !"
+not-enough-money: "Vous n'avez pas la somme requise : AMOUNT !"
+available-colors: "Les couleurs suivantes sont disponible : "
```
Comment (PyvesB) [inline src/main/resources/lang-FR.yml:39]:
```suggestion
available-colors: "Les couleurs suivantes sont disponibles : "
```

File: src/main/resources/lang-FR.yml [hunk 2]
```diff
@@ -1,57 +1,69 @@
...
-petmaster-command-free: Relâcher un animal.
-petmaster-command-setowner-hover: Vous pouvez seulement changer le propriétaire de vos propres bêtes !
-petmaster-command-disable-hover: Plugin désactivé jusqu'au prochain redémarrage ou /petm enable.
-petmaster-command-enable-hover: Plugin activé par défaut. A utiliser si /petm disable a été employé !
-petmaster-command-reload-hover: Recharger la plupart des paramètres dans config.yml et lang.yml.
-petmaster-command-free-hover: Vous pouvez seulement changer le propriétaire de vos propres bêtes !
-petmaster-command-info-hover: Des infos supplémentaires sur le plugin !
-petmaster-tip: '&lASTUCE&r &7&n&oCliquez&r &8ou &7&n&opassez votre curseur&r &8sur les commandes !'
+petmaster-command-setowner: "Changer le propriétaire d'un animal."
+petmaster-command-setcolor: "Définit la couleur des colliers de tout les animaux apprivoisé dans le futur."
```
Comment (PyvesB) [inline src/main/resources/lang-FR.yml:44]:
```suggestion
petmaster-command-setcolor: "Définit la couleur des colliers de tous les animaux apprivoisés dans le futur."
```

File: src/main/resources/lang-FR.yml [hunk 3]
```diff
@@ -1,57 +1,69 @@
...
+petmaster-command-free: "Relâcher un animal."
+petmaster-command-disable: "Désactiver PetMaster jusqu'au prochain redémarrage."
+petmaster-command-enable: "Activer PetMaster (si précédemment désactivé)."
+petmaster-command-reload: "Recharger la configuration du plugin."
+petmaster-command-info: "Afficher des informations à propos du plugin."
+petmaster-command-setowner-hover: "Vous pouvez seulement changer le propriétaire de vos propres animaux !"
+petmaster-command-setcolor-hover: "Actuellement les animaux apprivoisé ne sont pas affecté."
```
Comment (PyvesB) [inline src/main/resources/lang-FR.yml:51]:
```suggestion
petmaster-command-setcolor-hover: "Actuellement les animaux apprivoisés ne sont pas affectés."
```

File: src/main/resources/lang-FR.yml [hunk 4]
```diff
@@ -1,57 +1,69 @@
...
+petmaster-command-free-hover: "Vous pouvez seulement changer le propriétaire de vos propres animaux !"
+petmaster-command-disable-hover: "Plugin désactivé jusqu'au prochain redémarrage ou /petm enable."
+petmaster-command-enable-hover: "Plugin activé par défaut. A utiliser si /petm disable a été employé !"
+petmaster-command-reload-hover: "Recharger la plupart des paramètres dans config.yml et lang.yml."
+petmaster-command-info-hover: "Des infos supplémentaires sur le plugin et son incroyable auteur !"
```
Comment (PyvesB) [inline src/main/resources/lang-FR.yml:56]: ❤️

File: src/main/resources/lang-FR.yml [hunk 5]
```diff
@@ -1,57 +1,69 @@
 #============================================================#
 # +--------------------------------------------------------+ #
-# |          Pet Master 1.7 French language file           | #
+# |         Pet Master 1.12.2 French language file         | #
 # |                      By DarkPyves                      | #
 # +--------------------------------------------------------+ #
+# |                         Updade                         | #
```
Comment (PyvesB) [inline src/main/resources/lang-FR.yml:6]:
```suggestion
# |                         Update                         | #
```

**Review by PyvesB — COMMENTED** [review]
Merci beaucoup @Nogapra ! A few typos that need fixing, otherwise looks good to me. 😉

File: src/main/resources/lang-FR.yml (hunk 4 above)
Comment (Nogapra) [reply src/main/resources/lang-FR.yml:56]: ^^

**Comment by PyvesB** [comment]
Merci encore !

**Pull request merged**